R/helpers.R

```diff
@@ -532,6 +532,26 @@ cells_row_groups <- function(groups = TRUE) {
   cells
 }
 
+#' @noRd
+#' @export
+cells_group <- function(groups = TRUE) {
+
+  warning("The `cells_group()` function is deprecated and will soon be removed\n",
+          " * Use the `cells_row_groups()` function instead",
+          call. = FALSE)
+
+  # Capture expression for the `groups` argument
```

Comment:
Could the body of this function just be the deprecation warning, and then `cells_row_groups({{groups}})`?

Reply:
This was the first thing I tried but it resulted in an error (probably due to the unquoting). I did use `!!` on the arg value but it still didn’t work out.